packages/vuetify/src/components/VColorPicker/util/index.ts

```diff
@@ -54,9 +50,9 @@ function stripAlpha (color: any, stripAlpha: boolean) {
   return color
 }
 
-export function extractColor (color: HSVA, input: any) {
+export function extractColor (color: HSV, input: any) {
```

Comment:
small nitpick, should the color variable be hsv to match other naming conventions?